src/Resources/config/commands.yml

```diff
@@ -1,32 +1,26 @@
 services:
+    _defaults:
+        autoconfigure: true
```

Comment:
I guess it is pretty obvious what the option does in the `commands.yml` file, isn't it?